docker-compose.yml

```
@@ -1,57 +1,32 @@
-pgdata:
-  image: postgres:9.6
-  command: "true"
-  volumes:
-#Comentar la siguiente linea para corregir permisos en OSX  
-   - ./docker-data/pgdata:/var/lib/postgresql/data
+version: '3.7'
 
-   - ./docker-data/toba:/var/local/docker-data/toba
+services:
+  pg:
+    image: postgres:12
```

```suggestion
    image: postgres:14
```

O tenemos alguna limitante?